packages/next/server/next-dev-server.js

```diff
@@ -81,7 +81,7 @@ export default class DevServer extends Server {
               .filter(key => query[key] === undefined)
               .forEach(key =>
                 console.warn(
-                  `Url defines a query parameter '${key}' that is missing in exportPathMap`
+                  `Path '${path}' defines a query parameter '${key}' that is missing in exportPathMap`
```

Comment:
```suggestion
                  `Url '${path}' defines a query parameter '${key}' that is missing in exportPathMap`
```

It might be good to still say `url` to not confuse where the extra query value is coming from

Reply:
Sure, done